cdc/sink/codec/avro.go

```diff
@@ -312,6 +312,9 @@ func getAvroDataTypeFromColumn(col *model.Column) (interface{}, error) {
 	case mysql.TypeDouble:
 		return "double", nil
 	case mysql.TypeVarchar, mysql.TypeString, mysql.TypeVarString:
+		if col.Flag.IsBinary() {
```

Comment:
Could you add some unit tests?

Reply:
In the future, maybe we need a flag to control the behavior? Like https://debezium.io/documentation/reference/connectors/mysql.html#mysql-property-binary-handling-mode

Cc @leoppro